mkt/account/serializers.py

```diff
@@ -137,3 +138,12 @@ class UserSerializer(AccountSerializer):
     class Meta:
         model = UserProfile
         fields = ('display_name', 'resource_uri')
+
+
+class GroupsSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(source='pk', required=False, read_only=False)
```

Comment:
It should not be required, just adding it to the `fields` in `Meta` should be enough, but don't worry too much about it